src/pages/objkt-display/tabs/history.js

```diff
@@ -4,17 +4,35 @@ import { Primary } from '@components/button'
 import { walletPreview } from '@utils/string'
 import { getTimeAgo } from '@utils/time'
 import styles from '../styles.module.scss'
+import { BURN_ADDRESS } from '@constants'
 
 import { IconCache } from '@utils/with-icon'
-import TradeIcon from '@icons/trade'
-import MintedIcon from '@icons/minted'
-import SwapIcon from '@icons/swap'
+import { TradeIcon, MintedIcon, SwapIcon, BurnIcon } from '@icons'
+// import MintedIcon from '@icons/minted'
+// import SwapIcon from '@icons/swap'
+
+class OperationType {
```

Comment:
i think it would be simpler and easier to read to just use regular constants here:

```
const OPERATION_SWAP = 'SWAP';
const OPERATION_TRANSFER = 'TRANSFER';
const OPERATION_TRADE = 'TRADE';
```

Reply:
I was trying to do enums and discovered there is none in JS.
Can't we just use strings directly then?